server/handler.go

```diff
@@ -419,6 +420,34 @@ func esmHandler() rex.Handle {
 				return rex.Status(500, err.Error())
 			}
 
+			// Handle private package resources
+			if pkgInPrivateScope {
+				task := &BuildTask{
+					CdnOrigin: cdnOrigin,
+					Pkg:       reqPkg,
+					BuildArgs: BuildArgs{
+						alias:       map[string]string{},
+						deps:        PkgSlice{},
+						external:    newStringSet(),
+						treeShaking: newStringSet(),
+					},
+					Target: "raw",
+					stage:  "-",
+				}
+				c := buildQueue.Add(task, ctx.RemoteIP())
+				select {
+				case output := <-c.C:
+					if output.err != nil {
+						return rex.Status(500, "Raw file: "+output.err.Error())
+					}
+				case <-time.After(time.Minute):
+					buildQueue.RemoveConsumer(task, c)
+					return rex.Status(http.StatusRequestTimeout, "timeout, we are transforming the types hardly, please try again later!")
```

Comment:
can you please update the incorrect timeout message?

Reply:
sure,will update later